components/button/README.md

```diff
@@ -1,28 +1,50 @@
 # Buttons
 
-## d2l-button
-
-The `d2l-button` element can be used just like the native button element, but also supports the `primary` attribute for denoting the primary button.
-
-![Button](./screenshots/button.png?raw=true)
+A Button is used to communicate and perform an action.
 
 ```html
+<!-- docs: demo -->
 <script type="module">
   import '@brightspace-ui/core/components/button/button.js';
+  import '@brightspace-ui/core/components/button/button-subtle.js';
+  import '@brightspace-ui/core/components/button/button-icon.js';
 </script>
-<d2l-button>My Button</d2l-button>
+<d2l-button>Button</d2l-button>
+<d2l-button primary>Primary Button</d2l-button>
+<d2l-button-subtle text="Subtle Button" icon="tier1:gear"></d2l-button-subtle>
+<d2l-button-icon text="Icon Button" icon="tier1:gear"></d2l-button-icon>
 ```
 
-**Properties:**
+## Best Practices
+<!-- docs: start best practices -->
+<!-- docs: start dos -->
+* Use buttons to cause an action or launch a workflow
+* Keep button text short - see "Writing" guidelines
+<!-- docs: end dos -->
+
+<!-- docs: start donts -->
+* Don't use multiple Primary buttons - the primary action should always be clear and obvious
+* Avoid icon-only buttons with unfamiliar icons, only use these for familiar concepts
+* Don't use buttons for navigation, use a link instead
+* Don't open menus with buttons - use a dropdown instead
+<!-- docs: end donts -->
+<!-- docs: end best practices -->
+
+## Button [d2l-button]
+
+The `d2l-button` element can be used just like the native button element, but also supports the `primary` attribute for denoting the primary button.
 
-| Property | Type | Description |
```

Comment:
This deletes the properties tables. I can add a note for now that properties can be found within the code, since I believe we still aren't advertising Daylight. They can also still remain and we can wrap them in the <!-- docs: start hidden content --> comments. We would just probably then want to come back and remove that, then add a note about using Daylight, in order to only maintain the properties in the code.

Reply:
Right. If it's going to be a couple months before the Daylight Site is ready, it might be nice to leave these in for now and add the comments to hide them?